website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -174,10 +181,10 @@ default value in the `"/sys/config/ui"` [API endpoint](/vault/api-docs/system/co
 - `x_forwarded_for_authorized_addrs` `(string: <required-to-enable>)` –
   Specifies the list of source IP CIDRs for which an X-Forwarded-For header
   will be trusted. Comma-separated list or JSON array. This turns on
-  X-Forwarded-For support.  If for example Vault receives connections from the 
-  load balancer's IP of `1.2.3.4`, adding `1.2.3.4` to `x_forwarded_for_authorized_addrs` 
-  will result in the `remote_address` field in the audit log being populated with the 
-  connecting client's IP, for example `3.4.5.6`. Note this requires the load balancer 
+  X-Forwarded-For support.  If for example Vault receives connections from the
```

Comment:
I'm not going to add a suggestion here because it looks like you didn't actually change anything and git is just confused.